src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +85,15 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public String getPrintableString(Printable... printables){
```

Comment:
Consider using the StringJoiner class here.

Reference: https://docs.oracle.com/javase/8/docs/api/java/util/StringJoiner.html